server/__test__/getAllClientsForWorker.test.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable max-len */
+/* eslint-disable no-undef */
+// eslint-disable-next-line no-undef
```

Comment:
there should be a solution to those by setting up the eslint jest plugin correctly